crates/cargo-bsp/src/cargo_communication/request_handle.rs

```diff
@@ -31,9 +64,11 @@ impl RequestHandle {
         R::Params: CreateCommand + Send,
         R::Result: CargoResult,
     {
-        let cmd = params.create_command(root_path.into());
-        info!("Created command: {:?}", cmd);
-        let cargo_handle = CargoHandle::spawn(cmd)?;
+        let unit_graph_cmd = params.create_unit_graph_command(root_path.into());
+        info!("Created command: {:?}", unit_graph_cmd);
```

Comment:
czemu tylko tę komendę logujesz?

Reply:
Loguje komendy wtedy kiedy są spawnowane. Ta druga jest logowana w funkcji run_commands. A tworzę na raz dwie komendy, ponieważ params są przekazywane do request_actor i nie chciałam ich kopiować.

Reply:
to może lepiej przenieść ten log do ciała `CargoHandle::spawn`?

Reply:
Przeniesione